frontend/components/ReviewForm.js

```diff
@@ -48,7 +51,24 @@ class ReviewForm extends React.Component {
     });
   }
 
+  handleSubmit() {
+    // console.log(this.props.transactionState);
+    addTransaction(this.props.transactionState);
+    this.props.setTransactionState({
+      transactionItems: [],
+      transaction_date: new Date(),
+      staff_name: "Staff 1"
+    })
+  }
+
   render() {
+    let reviewItems = []
+
+    for (let i = 0; i < this.state.items.length; i++) {
+      reviewItems.push(<ReviewItem item={this.state.items[i]} onDelete={() => {
+        this.deleteItem(0);
```

Comment:
I think this should be this.deleteItem(i) rather than this.deleteItem(0), because otherwise clicking the delete button on any item would always delete the first review item